config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml

```diff
@@ -7,18 +7,22 @@ spec:
   networkAttachments:
   - ctlplane
   nodeTemplate:
+    networks:
```

Comment:
This is not required as it's using IPAM, see the `nodes` section. Only drop the ansible vars as you've done below.